Reaqtive/Core/Reaqtive.Linq/Reaqtive/Subscribable.cs

```diff
@@ -1091,6 +1091,24 @@ public static ISubscribable<TSource> FirstOrDefaultAsync<TSource>(this ISubscrib
 
         #endregion
 
+        #region IgnoreElements
+
+        /// <summary>
+        /// Suppresses all of the elements propagated by the source sequence, but still propagates its OnCompleted or OnError notification.
+        /// </summary>
+        /// <typeparam name="TSource">Type of the elements in the source sequence.</typeparam>
+        /// <param name="source">Source sequence whose elements to suppress.</param>
+        /// <returns>Observable sequence propagating only an OnCompleted or an OnError notification.</returns>
+        public static ISubscribable<TSource> IgnoreElements<TSource>(this ISubscribable<TSource> source)
+        {
+            if (source == null)
+                throw new ArgumentNullException(nameof(source));
+
+            return new Where<TSource>(source, x => false);
```

Comment:
It is fine to define one operator in terms of another, but it's a one-way street due to the way recovery works (it will persist query operator information and expects it to match upon recovery, i.e. `IgnoreElements` will have to always construct a `Where` node, because the checkpoint state of the query will expect to see a `Where` node). When doing something like this, we typically put a comment:

```csharp
//
// WARNING: Don't unfold the composition blindly; recovery of existing state won't be compatible.
//
```

However, for an operator as trivial as this, it may be better to just go ahead and implement the operator from scratch; at the very least, it will avoid invoking the `false`-returning predicates.